README.md

```diff
@@ -540,38 +535,52 @@ options:
   -t ENTRY_TYPE, --entry-type ENTRY_TYPE
                         Name of the type (from the CDDL) to interpret the data
                         as.
+  --default-max-qty DEFAULT_MAX_QTY, --dq DEFAULT_MAX_QTY
+                        Default maximum number of repetitions when no maximum
+                        is specified. It is only relevant when handling data
+                        that will be decoded by generated code. If omitted, a
+                        large number will be used.
+  --sanitize            Whether to sanitize/unsanitize CBOR-only values. When
```

Comment:
```suggestion
  --sanitize           Sanitize CBOR-only values. When
```

Isn't that correct, that if you set this you do that?
Or do you use this to 'unsanitize' as well?

Reply:
The script can convert both ways. Tried to explain this better.